x-pack/legacy/plugins/console_extensions/spec/overrides/ml.explain_data_frame_analytics.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
It might be good that someone from ML tested this in console to make sure it behave as expected?